cli/i18n/I18nUsageValidator.php

```diff
@@ -42,7 +42,7 @@ public function validate(): bool {
 		foreach ($this->reference as $file => $data) {
 			foreach ($data as $key => $value) {
 				$this->totalEntries++;
-				if (preg_match('/\._$/', $key) && in_array(preg_replace('/\._$/', '', $key), $this->code, true)) {
+				if (is_int(preg_match('/\._$/', $key)) && in_array(preg_replace('/\._$/', '', $key), $this->code, true)) {
```

Comment:
I would prefer something like the following (or `!= false`)

```suggestion
				if (preg_match('/\._$/', $key) === 1 && in_array(preg_replace('/\._$/', '', $key), $this->code, true)) {
```